### PR TITLE
Add django 1.8 support

### PR DESCRIPTION
**Pull request by Ian-Foote**

### CODE REVIEW

**Comment by meshy** [comment]
It's failing on a flake8 error -- want to fix that here or elsewhere?

**Comment by Ian-Foote** [comment]
@meshy Review/merge?

**Comment by Ian-Foote** [comment]
@adam-thomas @NoLogo Review/merge?

**Comment by adam-thomas** [comment]
LGTM

File: groups/tests/test_managers.py [hunk 1]
```diff
@@ -262,14 +262,8 @@ class MixedInManager(self.mixin, django_models.Manager):
...
-        class ProxyUser(get_user_model()):
-            """A proxy for the User model that adds in our manager."""
-            objects = MixedInManager()
-
-            class Meta:
-                proxy = True
-
-        self.model = ProxyUser
+        self.manager = MixedInManager()
+        self.manager.model = User
```
Comment (meshy) [inline groups/tests/test_managers.py]: Nice!

File: setup.cfg [hunk 2]
```diff
@@ -8,4 +8,4 @@ exclude = *migrations*,build*
 statistics = true
 application-import-names = groups
 import-order-style = google
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D203,D204
```
Comment (meshy) [inline setup.cfg]: D104?

**Comment by meshy** [comment]
LGTM

File: setup.cfg (hunk 2 above)
Comment (Ian-Foote) [inline setup.cfg]: [Missing docstring in public package](https://travis-ci.org/incuna/incuna-groups/jobs/85557658)

Comment (meshy) [inline setup.cfg]: Perhaps I'm being dim, but "Missing docstring in public package" doesn't really mean a lot to me. What it it really asking for here?

Comment (Ian-Foote) [inline setup.cfg]: [PEP 257:  A package may be documented in the module docstring of the `__init__.py` file in the package directory.](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)

Comment (meshy) [inline setup.cfg]: Aaah -- ok! Thanks! (Another silly error!)

**Pull request merged**